xml/System.Text.Json/JsonProperty.xml

```diff
@@ -211,9 +211,13 @@ This method is functionally equal to doing an ordinal comparison of `text` and <
         <Parameter Name="writer" Type="System.Text.Json.Utf8JsonWriter" />
       </Parameters>
       <Docs>
-        <param name="writer">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="writer">The writer to which to write the property.</param>
```

Comment:
This is difficult to read. Maybe we could rephrase it, or not use `writer`.
```suggestion
        <param name="writer">The output to which to write the property.</param>
```

Or maybe like this:
```suggestion
        <param name="writer">The output to write the property into.</param>
```